src/config.template.cjs

```diff
@@ -211,24 +211,25 @@ module.exports = {
 	 * "10min"
 	 * "2w"
 	 * "3 days"
+	 * "0" - this will search everything exactly once, never more.
 	 */
-	excludeOlder: undefined,
+	excludeOlder: "2 weeks",
 
 	/**
-	 * Exclude torrents which have been searched
-	 * more recently than this long ago.
+	 * Exclude torrents which have been searched more recently than this long ago.
+	 * Doesn't exclude previously failed searches.
 	 * Examples:
 	 * "10min"
 	 * "2w"
 	 * "3 days"
 	 */
-	excludeRecentSearch: undefined,
+	excludeRecentSearch: "3 days",
```

Comment:
Does anyone else think this setting should be more strict? By that I mean the options. For example, days can be `3 days` or `3d`. Should it be more opinionated with only one way of choosing periods?

Reply:
Good point. Just because ms supports it doesn't mean we have to tell people all the possible ways they can name their units. I'm in favor of the full length "5 minutes" style.

Reply:
Yup, I agree with using the longer form.